src/boot/boot.js

```diff
@@ -45,11 +45,19 @@ function bootHypothesisClient(doc, config) {
   // Register the URL of the sidebar app which the Hypothesis client should load.
   // The <link> tag is also used by browser extensions etc. to detect the
   // presence of the Hypothesis client on the page.
-  var baseUrl = doc.createElement('link');
-  baseUrl.rel = 'sidebar';
-  baseUrl.href = config.sidebarAppUrl;
-  baseUrl.type = 'application/annotator+html';
-  doc.head.appendChild(baseUrl);
+  var sidebarUrl = doc.createElement('link');
+  sidebarUrl.rel = 'sidebar';
```

Comment:
"hypothesis-sidebar" perhaps to reduce the risk of a conflict with other pages and for consistency with "hypothesis-client"?